### DISCUSSION TITLE
Cannot set Freq in Radio tab.

### DISCUSSION THREAD

**Discussion by 2e0wkw** [nakhonthai/ESP32APRS_T-TWR#23] [General]
When i try to enter 144800000 Hz i get an error message saying the number must be equal to or greater than 13333022312000.
I am not able to set the frequency.

**Comment by nmuntyanov** [comment]
Master is broken now

**Comment by 2e0wkw** [comment]
Should I install an older version ?On 31 Jan 2024, at 17:10, Mykola Muntianov ***@***.***> wrote:﻿
Master is broken now

**Comment by nmuntyanov** [comment]
> Should I install an older version ?

If u do not use SA868 Open Edition than you can checkout this commit  `8406fa278504d5b0352bcc37440f8bac9b8f6c8b`
`git checkout 8406fa278504d5b0352bcc37440f8bac9b8f6c8b`
and try to upload it

**Comment by nakhonthai** [comment]
Sorry for now code.I testing open edition version now from #17 . 
To use release firmware.
